packages/sdk/src/sdk/api/users/UsersApi.ts

```diff
@@ -543,29 +543,49 @@ export class UsersApi extends GeneratedUsersApi {
    * Share an encrypted email with a user
    */
   async shareEmail(params: EmailRequest, advancedOptions?: AdvancedOptions) {
-    const { emailOwnerUserId, receivingUserId, email, granteeUserIds } =
-      await parseParams('shareEmail', EmailSchema)(params)
+    const {
+      emailOwnerUserId,
+      receivingUserId,
+      email,
+      granteeUserIds,
+      initialEmailEncryptionUuid
+    } = await parseParams('shareEmail', EmailSchema)(params)
 
     let symmetricKey: Uint8Array
     // Get hashed IDs and validate
     const emailOwnerUserIdHash = encodeHashId(emailOwnerUserId)
     const receivingUserIdHash = encodeHashId(receivingUserId)
-    if (!emailOwnerUserIdHash || !receivingUserIdHash) {
+    const initialEmailEncryptionUuidHash = encodeHashId(
+      initialEmailEncryptionUuid
+    )
+    if (
+      !emailOwnerUserIdHash ||
+      !receivingUserIdHash ||
+      !initialEmailEncryptionUuidHash
```

Comment:
i think we can do `!(emailOwnerUserIdHash || initialEmailEncryptionUuidHash) || !receivingUserIdHash`? we don't need `initialEmailEncryptionUuidHash` in all possible cases necessarily

Reply:
we don't, but i did this because it's the only current use case and it makes the typing nicer. could always change it.